Automattic/newspack-plugin
# Feat/update transactional emails all templates

### All Submissions:






### Changes proposed in this Pull Request:



**PR 4 of 4**

On hold until https://github.com/Automattic/newspack-plugin/pull/3032, https://github.com/Automattic/newspack-plugin/pull/3033, and https://github.com/Automattic/newspack-plugin/pull/3034 are approved.

This PR finishes up the design updates for the transactional email work done in https://github.com/Automattic/newspack-plugin/pull/3032, https://github.com/Automattic/newspack-plugin/pull/3033, and https://github.com/Automattic/newspack-plugin/pull/3034 by updating the remaining transactional email templates with the ras-acc design:

Account verification:
![Screenshot 2024-04-05 at 20 14 13](https://github.com/Automattic/newspack-plugin/assets/17905991/40c0f408-fbbe-4f90-8ee5-7a072cbd8352)

Reset password:
![Screenshot 2024-04-05 at 20 13 05](https://github.com/Automattic/newspack-plugin/assets/17905991/4ef6fb4d-cf3f-4a6a-9ab4-4dfd917471f6)

Magic Link:
![Screenshot 2024-04-05 at 20 15 36](https://github.com/Automattic/newspack-plugin/assets/17905991/0bc8017e-4b50-4cd3-a9b8-76ef647eadb4)

Receipt:
![Screenshot 2024-04-05 at 20 16 13](https://github.com/Automattic/newspack-plugin/assets/17905991/cf043c63-d505-4b07-a6a0-dfe5a7dfbe24)

Account deletion:
![Screenshot 2024-04-05 at 20 12 25](https://github.com/Automattic/newspack-plugin/assets/17905991/bcde7452-1b07-4d29-aeb4-be7ab0c82455)

(OTP will have already been updated at this point)

### How to test the changes in this Pull Request:

1. Go to Newspack > Engagement > Show Advanced Settings > Transactional Email Content, then trash each of the emails shown here via the edit button (should be every one mentioned above except for the receipt template). This is so our default templates can be used for emails.
2. Go to Newspack > Reader Revenue > Emails, then trash the receipt email via the edit button.
3. Trigger each of the emails, then view them and verify they look as pictured above. You can trigger each email as follows:
  - Account verification - As a logged out user, sign up for a new account. Once logged in, go to My Account and select the `Send me a link` option to send a verification email
  - Password reset - Once verified, go back to My Account and select the Create a Password link
  - Receipt - Purchase a donation via the donation block. You will need to set up the block on any post or page to do this
  - Magic link -  I'm not actually sure tbh, since I can only ever get the OTP flow to work on my end. But you can test this one by sending yourself a test email via the Magic Link post edit page (Newspack > Engagement > Show Advanced Settings > Transactional Email Content, then edit next to Login Link). But it would be preferable to test it by triggering the email in the expected way if you know how to confirm the template tags are working
  - Account deletion - Go to My Account and select the Delete Account button

### Other information:
